### ISSUE TITLE
Broken TTS if switch day-night theme during review

### ISSUE THREAD

**Issue by tico-tico** [ankidroid/Anki-Android#5211]
1. Assume you use cards with TTS tag. Something like:

```
    front->
        <tts service="android" voice="ru_RU">
        <span class="vopros">{{Front}}</span>
        </tts>

    back->
        {{FrontSide}}
        
        <hr id=answer>
        
        <tts service="android" voice="ru_RU">
        <span class="otvet">{{Back}}</span>
        <tts>
```

2. Start review. Everything works fine.
3. Change day-night theme via switch on the side menu while you are on front side of card.
4. For reason unknown AnkiDroid starts reinit TTS.
5. Tap "Show answer"
6. AnkiDroid says "No TTS language available (ru_RU)"

I'm using Google TTS if it matters.

Reproduced on 2.9alpha58 on Android 4.4.4.

**Comment by mikehardy** [comment]
Fascinating - when you switch day/night mode we essentially re-init the app if I recall correctly. So I would expect TTS to possibly re-init as well. But obviously it should work again after. This makes me think there is some TTS resource that we aren't releasing correctly as the app is torn down, before it is rebuilt. Can you confirm TTS works again if you stop AnkiDroid and restart it? 

Also, just in case someone else starts tracking bugs, please include version numbers of device and AnkiDroid as you log these - TTS specifically may have behavior on Android 4.4.4 (your version if I remember correctly?) that is different than other Android versions.

Unrelated - a new alpha (2.9alpha58) was released yesterday with a fix (#5209) for the tangent issue you found in #5205 where you crashed on startup sometimes)

**Comment by tico-tico** [comment]
> Can you confirm TTS works again if you stop AnkiDroid and restart it?

Yep, even reviewing restart fixed the issue. I mean returning to the deck list and back to reviewing.

> Also, just in case someone else starts tracking bugs, please include version numbers of device and AnkiDroid as you log these - TTS specifically may have behavior on Android 4.4.4 (your version if I remember correctly?) that is different than other Android versions.

Oops. Sorry, I usually do. Fixed.

> Unrelated - a new alpha (2.9alpha58) was released yesterday with a fix (#5209) for the tangent issue you found in #5205 where you crashed on startup sometimes)

Yes, I know. GitHub finally has amazing and very useful "watch for releases" toggle.
